runtime-modules/storage/src/data_directory.rs

```diff
@@ -153,6 +162,41 @@ pub struct DataObjectInternal<
     pub ipfs_content_id: Vec<u8>,
 }
 
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Clone, Encode, Decode, PartialEq, Eq, Debug, Default)]
+pub struct Quota {
```

Comment:
I should have been more clear here that the limit is in terms of size, not in terms of number of objects sorry.

However, I think it actually could make sense to have both, so lets have both kinds of limits+usage in this struct: total size and number of objects.

Reply:
Done here https://github.com/Joystream/joystream/pull/2122/commits/ff989c45bcd0b849082ec16ae25b9adbab5cd13a#diff-5f5bdbc581be08dfecff6870d8fcf9b767904a351d254a70cf35267e5465589bR185